components/list/list-header.js

```diff
@@ -54,11 +60,11 @@ class ListHeader extends RtlMixin(LocalizeCoreElement(LitElement)) {
 			}
 			d2l-selection-summary {
 				flex: none;
-				margin-left: 0.9rem;
+				margin-inline-start: 0.9rem;
```

Comment:
Do we feel comfortable introducing CSS logical properties? I thought we were holding back a bit, although it's really just Safari 11 at this point that doesn't support it... and Safari 11 has all kinds of other issues.

Reply:
I switch it to be on the safe side.